cluster-provision/images/vm-image-builder/customize-image.sh

```diff
@@ -11,6 +11,22 @@ function cleanup() {
   virsh undefine "${DOMAIN_NAME}" || true
 }
 
+function wait_for_install_to_complete() {
+  count=0
+  retries=100
+  until [[ $(virsh list --all --name|grep $DOMAIN_NAME) == "" || $(virsh list --state-shutoff --name|grep $DOMAIN_NAME) =~ "" || $count -gt $retries ]]; do
```

Comment:
Why do you check against an empty regular expression in the second condition?

Reply:
Good catch. I'll compare against equals.

Reply:
Done.